configure.ac

```diff
@@ -430,10 +430,22 @@ AC_SUBST(LIBCTLGEOM_LIBS)
 
 ##############################################################################
 # check for libGDSII
-AC_CHECK_HEADER(libGDSII.h, [have_gdsii=maybe], [have_gdsii=no])
-if test $have_gdsii = maybe; then
-  AC_CHECK_LIB(GDSII, libGDSIIExists)
+AC_ARG_WITH(libGDSII, [AC_HELP_STRING([--without-libGDSII],[do not enable GDSII file support])], ok=$withval, ok=yes)
+LIBGDSII_LIBS=""
+if test "$ok" = "yes"; then
+    AC_CHECK_HEADER(libGDSII.h, [have_gdsii=maybe], [have_gdsii=no])
+    if test $have_gdsii = maybe; then
+        AC_CHECK_LIB(GDSII, libGDSIIExists, have_gdsii=yes, have_gdsii=no)
+    fi
+    if test "$have_gdsii" = "no"; then
+        AC_MSG_WARN([Disabling GDSII file support because libGDSII was not found])
+    else
+        AC_DEFINE(HAVE_LIBGDSII, 1, [Define if we have & link libGDSII])
```

Comment:
The `#define HAVE_LIBGDSII 1` and `LIBS+=-lGDSII` steps are automatically taken care of by the `AC_CHECK_LIB` macro.

Reply:
It only takes those steps if `action-if-found` is not present. I added my own `action-if-found` (have_gdsii=yes) while trying to add Ardavan's request for the `--without` flag. That made it necessary to take the extra steps manually (though there's probably a better way). I guess the real question is exactly what the `--with/--without` flags and default behavior should be. I never got an answer in #487.